business/istio_validations.go

```diff
@@ -13,8 +14,8 @@ import (
 )
 
 type IstioValidationsService struct {
-	k8s kubernetes.IstioClientInterface
-	ws  WorkloadService
+	k8s           kubernetes.IstioClientInterface
+	businessLayer *Layer
```

Comment:
Not sure if this breaks the pattern with the *Service struct to pass the clients.
I need to test locally to have a better look at it.

Reply:
I don't fully-follow you here. Do you mind elaborating a little bit here?
Do you think that I shouldn't remove `ws WorkloadService`?

Reply:
I'll let Lucas comment on this before merging, I kinda share his issue with this expose (and linking) businessLayer directly here instead of only the Service layers seeing each other.